src/components/ens-registration/IntroMarquee/IntroMarquee.tsx

```diff
@@ -52,20 +55,21 @@ export default function IntroMarquee() {
       const profiles: { [name: string]: string | undefined } = {};
       await Promise.all(
         ensIntroMarqueeNames.map(async name => {
-          const profile = await fetchProfile(name);
-          profiles[name] = profile?.records?.description;
+          const records = await fetchRecords(name);
```

Comment:
What do these lines do?

Reply:
we're only getting the description record in the line below so it doesn't make sense to call `fetchProfile`, which calls `fetchRecords` and additional fetch methods